### PR TITLE
Ldap fix and warning

### PR DESCRIPTION
**Pull request by adrianschroeter**

### CODE REVIEW

**Comment by adrianschroeter** [comment]
Needs to be backported to 2.7 (and 2.6?)

File: src/api/app/controllers/webui/user_controller.rb [hunk 1]
```diff
@@ -14,6 +14,8 @@ def index
 
   def logout
     logger.info "Logging out: #{session[:login]}"
+    key = "ldap_cache_userpasswd:" + session[:login]
+    Rails.cache.delete(key)
```
Comment (bgeuken) [inline src/api/app/controllers/webui/user_controller.rb]: I would prefer
```
Rails.cache.delete("ldap_cache_userpasswd:" + session[:login])
```

if key is only used once.

```
Rails.cache.delete("ldap_cache_userpasswd:#{session[:login]}")
```

Comment (bgeuken) [inline src/api/app/controllers/webui/user_controller.rb]: btw I couldn't find any ldap_cache_userpasswd string in our git sources. Are you sure this cache key exists?

Comment (adrianschroeter) [inline src/api/app/controllers/webui/user_controller.rb]: it is set inside of app/models/user_ldap_strategy.rb self.find_with_ldap

I will adapt the code.

Comment (bgeuken) [inline src/api/app/controllers/webui/user_controller.rb]: Oh! How did I miss that? -.-

**Pull request merged**